internal/system/agent/init.go

```diff
@@ -129,6 +129,7 @@ func initializeConfiguration(useRegistry bool, useProfile string) (*Configuratio
 	if err != nil {
 		return nil, err
 	}
+	configuration.Registry = config.OverwriteFromEnvironment(configuration.Registry)
```

Comment:
Consider renaming the func `OverwriteFromEnvironment()` to `OverrideFromEnvironment()`.

Reply:
Done.